### ISSUE TITLE
La license

### ISSUE THREAD

**Issue by Litarvan** [DuckHunt-discord/DuckHunt-Discord#6]
Creative Commons le disent, ce n'est pas fait pour les logiciels. En plus tu ne l'as marqué que dans le README ^^ Utilise la GPL par exemple  !

**Comment by Litarvan** [comment]
Ah, effectivement. Bah l'argument est simple, la Creative Commons n'est pas faite pour ça ils le disent, et la GPL devrait avoir les même conditions ^^

**Comment by Diagamma** [comment]
Nope, la GPL permet la vente, ce que la Creative Commons BY-NC-SA ne permet pas. C'est assez chaud de trouver une licence dans ce genre là pour les programmes.
